rdm/0006-profiles-and-flavours.md

```diff
@@ -0,0 +1,104 @@
+- Start Date: 2019-10-02
+- RFC PR: [#6](https://github.com/inveniosoftware/rfcs/pull/6)
+- Authors: Guillaume Viger
+
+# Customize created Invenio instance by choosing profile and optionally flavour
+
+## Summary
+
+When running the instance bootstrapping CLI command (`invenio-cli init`), the
+hoster should be given a choice to create an Integrated Library System (ILS) or
+Resource Data Management (RDM) "flavour" Invenio application. If RDM is chosen,
+there are further "profiles" to be chosen from, for instance "default" and
+"biomed". These choices pre-configure the application to varying levels.
+
+## Motivation
+
+> Why are we doing this? What user stories does it support? What is the expected
```

Comment:
For consistency, in other RFCs we do not include the questions. Shall we do it in all? or remove it from all

Reply:
Forgot it. Removed!